app/controllers/pipeline_work_in_progress_controller.rb

```diff
@@ -84,12 +104,35 @@ def mould_data_for_view(purposes, labware_records)
         state = decide_state(rec)
         next if state == 'cancelled'
 
-        output[rec.purpose.name] << { record: rec, state: state }
+        state_with_children = rec.has_children ? "#{state} (parent)" : state
+
+        output[rec.purpose.name] << { record: rec, state: state, state_with_children: state_with_children }
       end
     end
   end
 
   def decide_state(labware)
+    # TODO: the default of pending is a false assumption - see RVI cherrypick
```

Comment:
Needs a reference to ensure this is considered in future.

Reply:
A GitHub issue.  I cannot recount the number of times I've seen a TODO that I've had to try to dig through the Git commit history to find out why it's there.  Often the commit message is something like "Added TODO".  Then you ask around and no-one remembers why it's there and eventually you just delete it as unactionable.